shopfloor_reception/services/reception.py

```diff
@@ -304,8 +309,11 @@ def _scan_line__by_packaging(self, picking, barcode):
         search = self._actions_for("search")
         packaging = search.packaging_from_scan(barcode)
         if packaging:
-            move = picking.move_lines.filtered(
-                lambda m: packaging in m.product_id.packaging_ids
+            move = fields.first(
+                picking.move_lines.filtered(
+                    lambda m: packaging in m.product_id.packaging_ids
+                    and m.quantity_done < m.product_uom_qty
```

Comment:
use the force, use float_compare
```suggestion
                    and float_compare(m.quantity_done, m.product_uom_qty, rounding=m.product_uom.rounding) == -1
```